FordLabs/PeopleMover
# Update product endpoints to take in UUID

## Issue
Connects #334 

## What was done
- [x] Changed put, delete, and the post endpoints for product to include the space uuid and verified the product is mapped to the space
- [x] Upated product endpoint paths (i.e. 'api/space/{uuid}/products/' is the base url)

## How to test
1. App should be able to create, update, and delete products as usual (nothing should have changed functionality-wise for the user)
